pkgs/top-level/python-packages.nix

```diff
@@ -7835,6 +7895,24 @@ in modules // {
     };
   };
 
+  flask-pymongo = buildPythonPackage rec {
+    name = "Flask-PyMongo-${version}";
+    version = "0.3.1";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/F/Flask-PyMongo/${name}.tar.gz";
+      sha256 = "0305qngvjrjyyabf8gxqgqvd9ffh00gr5yfrjf4nncr2my9svbyd";
+    };
+
+    pythonPath = with self; [ flask pymongo_2_9_1 ];
+
+    meta = {
+      homepage = "http://flask-pymongo.readthedocs.org/";
+      description = "PyMongo support for Flask applications";
+      license = licenses.bsd;
```

Comment:
this license does not exist, `bsd2`, `bsd3` or `bsdOriginal` if I am not mistaken